### DISCUSSION TITLE
When a threat prevention event is logged it spans on multi-lines while the bahavior is different for the other logged events

### DISCUSSION THREAD

**Discussion by adamlahbib** [openappsec/openappsec#239] [Feature Requests]
![image](https://github.com/openappsec/openappsec/assets/37028131/c39479fc-0ccf-45ff-a9aa-35a6e04df6ca)

This is a preview of what I am talking about, and while viewing the logs in Google Cloud Logging that log item would span across multiple lines making it difficult to filter based on it.
![image](https://github.com/openappsec/openappsec/assets/37028131/4cefa994-a1b4-4c89-8fcf-cfb524f1b910)

**Comment by orianelou** [comment]
Hi @adamlahbib, 

Thank you for submitting this PR, this is behavior is by design, as we wish to decrease the size the logs are taking, but we'll look into allowing multiline in another way.
